sql/core/src/test/scala/org/apache/spark/sql/execution/streaming/state/StatePartitionAllColumnFamiliesWriterSuite.scala

```diff
@@ -329,15 +389,42 @@ class StatePartitionAllColumnFamiliesWriterSuite extends StateDataSourceTestBase
           performRoundTripTest(
             sourceDir.getAbsolutePath,
             targetDir.getAbsolutePath,
-            keySchema,
-            valueSchema,
-            keyStateEncoderSpec
+            createSingleColumnFamilySchemaMap(keySchema, valueSchema, keyStateEncoderSpec)
           )
         }
       }
     }
   }
 
+  private val keyToNumValuesColFamilyNames = Seq("left-keyToNumValues", "right-keyToNumValues")
```

Comment:
Many of these are duplicate of what we do in the multi-cf reader suite too. Should we move them to a common test util? Or are they different?